R/layers2traces.R

```diff
@@ -951,6 +959,10 @@ hover_on <- function(data) {
 
 # make trace with errorbars
 make_error <- function(data, params, xy = "x") {
+  # if xy is NULL: set xy to mean of xy_min and xy_max
+  if (is.null(data[[xy]])) {
+    data[[xy]] <- (data[[paste0(xy, "min")]] + data[[paste0(xy, "max")]]) / 2  
+  }
```

Comment:
```suggestion
  data[[xy]] <- data[[xy]] %||% (data[[paste0(xy, "min")]] + data[[paste0(xy, "max")]]) / 2  
```

Reply:
We need another pair of parentheses on the RHS:
```suggestion
data[[xy]] <- data[[xy]] %||% ((data[[paste0(xy, "min")]] + data[[paste0(xy, "max")]]) / 2)
```